src/gluonts/torch/model/estimator.py

```diff
@@ -147,16 +146,7 @@ def train_model(
         checkpoint = pl.callbacks.ModelCheckpoint(
             monitor=monitor, mode="min", verbose=True
         )
-        early_stopping = pl.callbacks.EarlyStopping(
-            monitor=monitor,
-            mode="min",
-            verbose=True,
-            patience=10,
-            check_on_train_epoch_end=validation_data is None,
-        )
-        trainer = pl.Trainer(
-            callbacks=[checkpoint, early_stopping], **self.trainer_kwargs
-        )
+        trainer = pl.Trainer(callbacks=[checkpoint], **self.trainer_kwargs)
```

Comment:
Checkpointing is still overridden by passed callbacks, but not too important for this PR.

Reply:
well it actually is, because who knows what model will be loaded at the end... so I'd say let's keep the default list of callbacks maybe